pkg/project/runtime/runtime.go

```diff
@@ -55,7 +55,12 @@ func customBuildContext(entrypointFilePath string, dockerfilePath string, buildA
 		return nil, err
 	}
 
-	// Get the ignore file contents
+	// TODO Get the ignore file contents
```

Comment:
How difficult is this to add? I think we were avoiding TODO statements after v1

Reply:
I can remove this, it was already a TODO just without the comment